setup.cfg

```
@@ -1,10 +1,89 @@
+[metadata]
+name = python-lsp-server
+author = Python Language Server Contributors
+description = Python Language Server for the Language Server Protocol
+url = https://github.com/python-lsp/python-lsp-server
+long_description = file: README.md
+long_description_content_type = text/markdown
+
+[options]
+packages = find:
+python_requires = >=3.6
```

```suggestion
python_requires = >=3.7
```

This changed in PR #130.

---

Fixed.